install.md

```diff
@@ -0,0 +1,108 @@
+<!-- vim-markdown-toc GFM -->
+
+* [Nmstate Installation Guide](#nmstate-installation-guide)
+    * [RPM based](#rpm-based)
+        * [Stable release](#stable-release)
+        * [Developer Branch](#developer-branch)
+    * [PyPI/pip](#pypipip)
+        * [Stable Release](#stable-release-1)
+        * [Developer Branch](#developer-branch-1)
+    * [setup.py](#setuppy)
+        * [Stable Release](#stable-release-2)
+        * [Developer Branch](#developer-branch-2)
+
+<!-- vim-markdown-toc -->
+
+# Nmstate Installation Guide
+
+## RPM based
+
+### Stable release
+
+The nmstate is in stable repo of Fedora 28+ and EPEL7-testing repo, you may
+install it using below commands.
+
+ * Fedora 28+
+```bash
+sudo dnf install nmstate -y
+```
+
+ * RHEL 7 using EPEL7-testing.
+
+```bash
+sudo yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+sudo yum-config-manager --enable epel-testing
+sudo yum install -y nmstate
```

Comment:
The official docs for EPEL recommend to also enable the optional and extras repositories - maybe it should be mentioned as well.